test/extended/util/cli.go

```diff
@@ -218,6 +221,12 @@ func (c *CLI) Template(t string) *CLI {
 	return c
 }
 
+// InputString adds expected input to the command
+func (c *CLI) InputString(input string) *CLI {
```

Comment:
Thanks for this :-) We can get rid of creating temporary files when processing template.